docs/references/strategies/system/rpm/rpm-container.md

```diff
@@ -4,6 +4,22 @@ The RedHat Package Manager (rpm).
 
 > This analysis is only executed when container scanning.
 
+## Important Limitations
+
+**License Identification**: RPM package detection is supported, but license information extraction has limitations:
+
+- **Full Support**: Alpine (APK) and Debian (DPKG) packages include complete license information.
+- **Limited Support**: RHEL and Oracle Linux (OL) RPM packages are detected but may appear as "unlicensed" because:
+  - License information is not currently extracted from RPM package databases
+  - Oracle Linux EPEL repositories are not automatically recognized
```

Comment:
Oracle Linux should be supported. I don't see any open issues for Oracle and do see a number of closed ones delivering support.